### ISSUE TITLE
Multiworld scoreboard doesn't work

### ISSUE THREAD

**Issue by xS91D** [ArchonKlaus/AdvancedScoreboard#17]
Multiworld scoreboard doesn't work

**Comment by samupozzo** [comment]
yes, the method is deprecated, use the v2.0 branch: https://github.com/SamuelPozzobon/AdvancedScoreboard/tree/2.0

**Comment by xS91D** [comment]
why deprecated ?

**Comment by samupozzo** [comment]
I need to rewrite the entire multiworld system, this plugins was intended to run in PM3, the API has changed too much and the level system is changed. I'll provide a solution ASAP.

**Comment by samupozzo** [comment]
I merged the first alpha of 2.0 version in the main branch, you can download the src here: https://github.com/SamuelPozzobon/AdvancedScoreboard (the pre-release is old, I need to republish it).
